ocaml/idl/datamodel_repository.ml

```diff
@@ -73,6 +82,19 @@ let introduce =
     ~allowed_roles:(_R_POOL_OP ++ _R_CLIENT_CERT)
     ()
 
+let introduce_bundle =
+  call ~name:"introduce_bundle" ~in_oss_since:None ~lifecycle:[]
+    ~doc:"Add the configuration for a new bundle repository"
+    ~params:
+      [
+        (String, "name_label", "The name of the repository")
+      ; (String, "name_description", "The description of the repository")
+      ; (String, "gpgkey_path", "The GPG public key file name")
```

Comment:
I think this can be removed. It's a remnant of an aborted feature a few years ago.